home.js

```diff
@@ -10,4 +10,195 @@ window.onload = function () {
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Profesional Art',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Data Dashboard',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
+    headerName: 'Website Protfolio',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required has been the industrys standard',
+    image1: './assets/Placeholder44b.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+];
+
+function showPopup() {
+  document.querySelector('.multi-cont').style.display = 'flex';
+}
+
+/* ----------------- Building Popup (Mobile) ---------------- */
+document.addEventListener('DOMContentLoaded', (event) => {
+  const popupSection = document.querySelector('.multi-cont');
+  const popupCont = document.createElement('div');
+  popupCont.className = 'multi-cont2';
+  popupSection.append(popupCont);
+  const closeSign = document.createElement('img');
+  closeSign.classList.add('closesign');
+  closeSign.src = './assets/Icon.png';
+  closeSign.alt = 'closesign';
+  closeSign.onclick = hidePopup;
+  const popupTitle = document.createElement('h3');
+  popupTitle.className = 'multi-stories';
+  const popupUl = document.createElement('ul');
+  popupUl.className = 'ul-tech1';
+  const popupImg = document.createElement('img');
+  popupImg.alt = 'project';
+  popupImg.className = 'image1';
+  const popupDescription = document.createElement('p');
+  popupDescription.className = 'multitext';
+  const liveButton = document.createElement('button');
+  liveButton.innerText = 'See Live';
+  liveButton.className = 'btn-pop1 hover';
+  const sourceButton = document.createElement('button');
+  sourceButton.innerText = 'See Source';
+  sourceButton.className = 'btn-pop2 hover';
+  popupCont.append(
+    closeSign,
+    popupTitle,
+    popupUl,
+    popupImg,
+    popupDescription,
+    liveButton,
+    sourceButton,
+  );
```

Comment:
- [ ] In desktop version, the content is overflowing the popup and we still can't see the content (the buttons) like the previous reviewer mentioned, kindly fix the issue 👍 

[ Optional ]
- [ ] It's better if we can't scroll in the website when the popup window is open, that would make the experience nicer and smoother 💯 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/202760589-57c4206d-76bf-4f12-9005-a12f65f5f5c4.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/202760673-d06471f9-3c95-413d-9841-6f5cd67eadfa.png)